src/utils/parse-ids-and-names.ts

```diff
@@ -0,0 +1,8 @@
+export function parseIdsAndNames(args: string[]): { ids: number[]; names: string[] } {
+    const allArgs = args.map((a) => a.split(",")).flat();
```

Comment:
why additionally (to space) split by comma?

And why not move the id matching into scriptsMatchingPattern? (if `names` argument is renamed to something like `patterns` and can contain names, groups and ids that would a logic and simple approach)

Reply:
> why additionally (to space) split by comma?
because I used it in all my tests "wrong" with the comma until the end I checked that it can also be used with spaces :sweat_smile:. Then I thought it would be a good idea to support both, what do you think?


> if `names` argument is renamed to something like `patterns` and can contain names, groups and ids that would a logic and simple approach
Good point, I'll refactor it :+1:

Reply:
yes, it won't do any harm.